README.md

```diff
@@ -342,6 +342,30 @@ While several Ruby books suggest the first style, the second is much more promin
 
 Refactoring is even better. It's worth looking hard at any code that explicitly checks types.
 
+- For single-line lambdas, `lambda` or `->` may be used.
+
+```ruby
+  scope :example, -> { where(:a => b) }
+
+  # Or
+
+  scope :example, lambda { where(:a => b) }
+```
+
+- For multi-line lambdas, prefer `lambda do ... end`
+
+```ruby
+  # bad
+  scope :long_example, lambda {
+    # ...
+  }
+
+  # good
+  scope :long_example, (lambda do
```

Comment:
http://devblog.avdi.org/2011/07/26/the-procedurefunction-block-convention-in-ruby/ might be an interesting read to distill into our style guide. It's mostly (mostly!) the way I write blocks. (Although I simplify it usually to `do/end` for multiline, unless I'm chaining or `{}` stops me using parens.)